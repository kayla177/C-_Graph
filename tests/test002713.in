ENTITY J5IFNZMP name_J5IF institution
ENTITY PVYDYRR8 name_PVYD paper
ENTITY FD3D8EUL name_FD3D institution
ENTITY J7UGPE6A name_J7UG paper
ENTITY JP82BULI name_JP82 institution
ENTITY IH76NS0Q name_IH76 institution
ENTITY B96LO34R name_B96L author
ENTITY I0G9TBJD name_I0G9 conference
ENTITY PYCKBU6Q name_PYCK paper
ENTITY ETFP6QBI name_ETFP institution
RELATIONSHIP J7UGPE6A collaborated_with B96LO34R 7.5
RELATIONSHIP JP82BULI cited_by J5IFNZMP 0.7
RELATIONSHIP PVYDYRR8 cited_by I0G9TBJD 1.4
RELATIONSHIP PYCKBU6Q collaborated_with J5IFNZMP 6.9
RELATIONSHIP FD3D8EUL presented_at I0G9TBJD 4.8
RELATIONSHIP ETFP6QBI affiliated_with B96LO34R 2.7
RELATIONSHIP IH76NS0Q published_in ETFP6QBI 5.6
RELATIONSHIP ETFP6QBI affiliated_with J7UGPE6A 3.0
RELATIONSHIP JP82BULI published_in PVYDYRR8 6.2
RELATIONSHIP FD3D8EUL collaborated_with ETFP6QBI 7.4
RELATIONSHIP ETFP6QBI published_in PYCKBU6Q 3.1
RELATIONSHIP B96LO34R presented_at PVYDYRR8 8.0
RELATIONSHIP PYCKBU6Q cited_by I0G9TBJD 3.4
RELATIONSHIP J5IFNZMP presented_at B96LO34R 4.7
RELATIONSHIP JP82BULI cited_by FD3D8EUL 0.5
HIGHEST
PATH PYCKBU6Q J5IFNZMP
DELETE B96LO34R
PRINT PVYDYRR8
PATH JP82BULI ETFP6QBI
DELETE PVYDYRR8
PRINT FD3D8EUL
PRINT PYCKBU6Q
PATH JP82BULI I0G9TBJD
HIGHEST
EXIT
