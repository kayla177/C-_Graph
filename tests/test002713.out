success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
JP82BULI PYCKBU6Q 25.8
PYCKBU6Q J5IFNZMP 6.9
success
I0G9TBJD JP82BULI
JP82BULI PVYDYRR8 I0G9TBJD FD3D8EUL ETFP6QBI 19.8
success
I0G9TBJD ETFP6QBI JP82BULI
J5IFNZMP ETFP6QBI I0G9TBJD
JP82BULI J5IFNZMP PYCKBU6Q I0G9TBJD 11
IH76NS0Q I0G9TBJD 17.8
